block_model.rb

```diff
@@ -0,0 +1,73 @@
+class BlockModel
```

Comment:
Its necesary to comment the code for others members to understand what are doing the function  and it is prefeerable to use a good var names also.